package.json

```diff
@@ -45,62 +52,57 @@
       ]
     ]
   },
-  "config": {
-    "ghooks": {
-      "pre-commit": "cat src/scripts/**/*.js | jscs --preset=airbnb; npm run lint; npm test"
-    }
-  },
   "scripts": {
-    "serve": "http-server -c-1 -p 9966",
+    "serve": "npx http-server -c-1 -p 9966",
```

Comment:
Do we need a `npx` dev dependency?

Reply:
npx is an automatic part of npm 5+ in all cases where I've used npm5+, including docker.

Reply:
My understanding is that it comes with npm v5 (https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b)

Reply:
Ok awesome 👍 LGTM in that case